fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/NotificationRobot.kt

```diff
@@ -28,35 +30,48 @@ class NotificationRobot {
         var notificationFound = mDevice.findObject(notification).waitForExists(waitingTime)
 
         while (!notificationFound) {
+            Log.i(TAG, "verifySystemNotificationExists: While block")
```

Comment:
I don't think we need this logged, as the proceeding messages will explicity imply that we're already in here